### ISSUE TITLE
Modula Image Alt Text Bug

### ISSUE THREAD

**Issue by alexjpanagis** [MachoThemes/modula-lite#222]
The Modula Alt Text is just pulled from the image caption when an alt text is not specified, so it cannot be left empty.
